src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/DescriptorReferenceValidation.feature.disabled

```diff
@@ -0,0 +1,82 @@
+                  | 0063560B5AC641DBBCCE4B121C5CA37B | State                  |
+
+
+        @ignore
+        Scenario: User CAN'T create a resource when reference doesn't exist
+             When client sends a POST request to "/ed-fi/localEducationAgencies" with body
+                  """
+                  {
+                        "localEducationAgencyId": 25590100,
+                        "nameOfInstitution": "Grand Institution Test",
+                        "categories": [
+                        {
+                        "educationOrganizationCategoryDescriptor": "uri://ed-fi.org/EducationOrganizationCategoryDescriptor"
+                        }
+                        ],
+                        "localEducationAgencyCategoryDescriptor": "uri://ed-fi.org/LocalEducationAgencyCategoryDescriptor"
+                  }
+                  """
+             Then it should respond with 400
+              And the response body is
+                  """
+                  {"detail":"Data validation failed. See 'validationErrors' for details.","type":"urn:ed-fi:api:bad-request:data","title":"Data Validation Failed","status":400,"correlationId":null,"validationErrors":{"$.localEducationAgencyCategoryDescriptor":["LocalEducationAgencyCategoryDescriptor value 'uri://ed-fi.org/LocalEducationAgencyCategoryDescriptor' is not a valid value for use as a descriptor (a '#' is expected to delineate the namespace and code value portions)."],"$.categories[0].educationOrganizationCategoryDescriptor":["EducationOrganizationCategoryDescriptor value 'uri://ed-fi.org/EducationOrganizationCategoryDescriptor' is not a valid value for use as a descriptor (a '#' is expected to delineate the namespace and code value portions)."]}}
+                  """
+
+        @ignore
+        Scenario: User can POST a resource when reference exists
+             When client sends a POST request to "/ed-fi/localEducationAgencies" with body
+                  """
+                  {
+                    "localEducationAgencyId": 25590100,
+                    "nameOfInstitution": "Grand Institution Test",
+                    "categories": [
+                        {
+                            "educationOrganizationCategoryDescriptor": "uri://ed-fi.org/EducationOrganizationCategoryDescriptor#Education+Service+Center"
+                        }
+                    ],
+                    "localEducationAgencyCategoryDescriptor": "uri://ed-fi.org/LocalEducationAgencyCategoryDescriptor#Public+school+district+part+of+a+supervisory+union"
+                  }
+                  """
+             Then it should respond with 201
+
+
+        @ignore
+        Scenario: User can UPDATE a resource when reference exists
+             When client sends a PUT request to "/ed-fi/students/fafbbd2024664ee786c05e899fbfe9bc" with body
+                  """
+                  {
+                    "studentUniqueId": "604824",
```

Comment:
As discussed in the retrospective, please change line 53 to have "/ed-fi/students/{id}" and add `"id": "{id}",` above line 56.

Reply:
Completed. 
I executed this test twice with "id" parameter and without it. And this field is not required as part of the body of the PUT request

Reply:
In the ODS/API there is a known bug - `id` _should be_ required. This is a requirement for the DMS. Please add `"id": "{id}" above  `"studentUnqueId"`.

Reply:
Done